swap_meet/clothing.py

```diff
@@ -1,2 +1,19 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self,id = None, fabric = "Unknown", condition = 0):
+        super().__init__(id, condition)
+        self.fabric = fabric
+        self.condition = condition
```

Comment:
Since the parent class keeps track of condition, you do not need to keep track of it here.